rest/v10/index.ts

```diff
@@ -461,6 +462,22 @@ export const Routes = {
 		return `/guilds/${guildId}/templates/${code}` as const;
 	},
 
+	/**
+	 * Route for:
+	 * - GET `/channels/{channel.id}/poll/{message.id}/answers/{answer_id}`
+	 */
+	pollAnswerVoters(channelId: Snowflake, messageId: Snowflake, answerId: string) {
+		return `/channels/${channelId}/poll/${messageId}/answers/${answerId}` as const;
```

Comment:
The routes were correct before 2c4b8acb2ae81a4afd4678c2597cb1f06e469614. It's `polls`, not `poll`